src/scss/components/_form.scss

```diff
@@ -59,18 +115,21 @@
   }
 
   &__label {
-    display: inline-block;
+    display: block;
     margin-bottom: 6px;
     color: get-color((color: blackNeutral, opacity: .9));
-    font-size: 15px;
+    font-size: 13px;
     font-weight: 700;
   }
 
   &__label-link {
     margin-left: 10px;
+    float: right;
```

Comment:
I ran the project here, and with a large label, this `form__label-link` seems break, like this sample:

![screen shot 2017-04-24 at 9 26 56 pm](https://cloud.githubusercontent.com/assets/1279783/25363925/fae34158-2934-11e7-8335-cc47d2788341.png)

In this case, do you think that `display: block` and `text-align: right` will work better? See the sample above:

![screen shot 2017-04-24 at 9 31 34 pm](https://cloud.githubusercontent.com/assets/1279783/25363988/7720fa08-2935-11e7-827f-3911e7b28678.png)

I think that is a decision to designer...